### ISSUE TITLE
Plugin fails to download Azure IP's

### ISSUE THREAD

**Issue by tsteur** [matomo-org/plugin-TrackingSpamPrevention#79]
Getting this error:

> Scheduler: Error Expected download URL for Azure IP ranges to start with HTTP but it does not. It is: .json for task 'Piwik\Plugins\TrackingSpamPrevention\Tasks.updateBlockedIpRanges'

This used to work but now started failing.

**Comment by snake14** [comment]
@tsteur I enabled TrackingSpamPrevention and enabled blocking cloud IP addresses. I then saw in the comment block of that task that it could be manually run using `./console core:run-scheduled-tasks --force "Piwik\Plugins\TrackingSpamPrevention\Tasks.updateBlockedIpRanges"`, so I tried that. It completed without any errors.

Any recommendations on what else I could try to reproduce the error?

**Comment by tsteur** [comment]
Thanks @snake14 I'll close it again for now and will reopen should it happen again tomorrow

**Comment by tsteur** [comment]
Actually @snake14 I believe the "block cloud" setting also needs to be enabled. Is that the case? You find it in Administration -> General Settings
<img width="859" alt="image" src="https://user-images.githubusercontent.com/273120/204930298-f168aab3-8204-4885-86c5-692edb2d3a93.png">

**Comment by snake14** [comment]
@tsteur I'm pretty sure I had the right setting enabled before I ran the console command. Is this the one you're talking about?
![image](https://user-images.githubusercontent.com/11394099/204941370-c885a0c0-a530-41b5-92eb-4a46e00d7060.png)

Edit: I think it's the right one. I ran the command again with the debugger and [line 41 in the task](https://github.com/matomo-org/plugin-TrackingSpamPrevention/blob/4.x-dev/Tasks.php#L41) evaluated to true and `updateBlockedIpRanges()` was executed on line 42.
